README.md

```diff
@@ -2,55 +2,35 @@
 
 # Project Name
 
-> Description the project.
+NIGERIA FASHION SKILL ACQUISITION SHOW.
 
+> This is my Capstone project for the first module in microverse, it is a static website for a Nigeria fashion skill acquisition show
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
 
-## Live Demo (if available)
+## [Live Demo Link]
 
-[Live Demo Link](https://livedemo.com)
+(https://sonyco-4u.github.io/Skill-Acquiziation-Show/)
```

Comment:
- [ ] Good job so far 👏. As suggested by the previous reviewer, It looks like your Live Demo link is not working. As in the project requirements, you are expected to deploy your website using GitHub pages because visitors will easily check your Live demo
![image](https://user-images.githubusercontent.com/90222110/192087682-ca24bfdf-58c0-465b-b6ce-4c597dcae9f1.png)

Currently, this is what is being displayed when I click on the above link 👇 
![image](https://user-images.githubusercontent.com/90222110/192087748-fc99ce51-61fa-49f0-b53a-3841b5016825.png)

Please deploy again your website and make sure that the live demo link is working 😉 you can check out [this article](https://www.freecodecamp.org/news/publish-your-website-netlify-github/). It may be helpful